### DISCUSSION TITLE
Can i get acceleration states out of the sim when using the ```create_smooth_transition_models``` function?

### DISCUSSION THREAD

**Discussion by faultfactory** [dstl/Stone-Soup#1053] [Q&A]
Hello, 
I'd like to know if i can generate states that have acceleration populated in them when using the ```create_smooth_transition_models``` function 
When i feed the initial state is tells me there's a mismatch so using the initial state clearly isn't correct. I'd really just need this on the output states for the platforms I'm simulating. Is there  something I'm missing?

**Comment by sdhiscocks** [comment]
No, not currently. The function generates a sequence of models for 2d position/velocity states at the moment. However, the models do calculate the accelerations required (`ax` and `ay` values), however not for the turn model that's used. It shouldn't be too difficult to modify the code to (optionally) add acceleration to the state space for the Point2Point models (see `StateVector` returns in `function` method), but need a bit more work to handle it with `KnowTurnRate` model.

You may also want to consider the `ConstantJerkSimulator`, which could be extended to support acceleration in the state vector as well.
